spec/unit/provider/remote_directory_spec.rb

```diff
@@ -79,7 +79,7 @@ class Chef::CFCCheck
     end
 
     it "configures access control on intermediate directorys" do
-      directory_resource = @provider.send(:resource_for_directory, File.join(Dir.tmpdir, "intermediate_dir"))
+      directory_resource = @provider.send(:directory_resource, File.join(Dir.tmpdir, "intermediate_dir"))
```

Comment:
Yeah, I still can't decide how I feel about that, but I just left that how I found it.   Also kinda smells like the specs might need some refactoring to test the public api instead of the inner hooks, but that's very much out of scope.